backend/src/types/RequestTypes.ts

```diff
@@ -0,0 +1,28 @@
+import { UserModel } from '../models/UserModel';
+
+export interface LoginRequest {
+    credential: string;
+}
+
+export interface DevLoginRequest {
+    id: string;
+    name?: string;
+    profileUrl?: string;
+}
+
+export interface AuthenticatedRequest {
+    user: UserModel;
+}
+
+export interface MeetingRequest {
+    name: string;
+    time: string;
+    invited: {
+        userId: string[];
+        roleName: string[];
+    };
+    where: string;
+    attended: string[];
+    absent: Map<String, String>;
+    description?: string;
+}
```

Comment:
I'm not sure if this is possible, but maybe try something like:
```ts
interface MeetingRequest extends Omit<MeetingModel, 'time' | '_id'> {
    time: string;
}
```

So that if we add more fields to `MeetingModel` it automatically gets added to this, and you can exclude the ones like time, which will be a `string` in the request rather than a `Date`.

Reply:
I'll give it a go, and if I run into any problems, then I'll check in with you guys.